e2e/pnpm_lockfiles/base/package.json

```diff
@@ -3,6 +3,9 @@
     "private": true,
     "dependencies": {
         "@aspect-test/a": "^5.0.2",
+        "debug": "ngokevin/debug#9742c5f383a6f8046241920156236ade8ec30d53",
+        "hello": "https://gitpkg.vercel.app/EqualMa/gitpkg-hello/packages/hello",
+        "jsonify": "https://github.com/aspect-build/test-packages/releases/download/0.0.0/@foo-jsonify-0.0.0.tgz",
```

Comment:
Are there more from `npm/private/test/package.json` that should be pulled into `e2e/pnpm_lockfiles`?

Reply:
Yeah there probably are. I need to get this PR actually working before I add much more though.

The `debug` and `hello` ones are unique cases (odd git syntax, https syntax). The `jsonify` one is also plain http but has a more interesting URL which requires escaping.